scripts/dotnet-cli-build/runtime-graph.json

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
This is the legit runtime graph, I think we'll need to find another way to load it instead of checking it in, but this might work.